test/util_test.py

```diff
@@ -69,6 +69,29 @@ def test_get_instance_region_failure(self):
         # Verify a warning is thrown
         mock_logger.warn.assert_called_once_with('get_instance_region failed to get the local instance region')
 
+    def test_setup_host_with_no_domain(self):
+        """
+        setup_hosts correctly adds default domain to hosts with no domain
+        """
+        mock_host_list = ['ops-dev004', 'ops-dev003', 'ops-dev005']
+        appended_hosts = setup_hosts(mock_host_list)
+        self.assertEquals(['ops-dev004.krxd.net', 'ops-dev003.krxd.net', 'ops-dev005.krxd.net'], appended_hosts)
```

Comment:
You can link the two list up using list comprehension. I don't want to update two lists if I want to change the test to use `ops-dev100` instead of `ops-dev004`.